app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,69 @@
+class FoodsController < ApplicationController
+  before_action :set_food, only: %i[show edit update destroy]
+  load_and_authorize_resource
+
+  # GET /foods or /foods.json
+  def index
+    @foods = Food.all
+  end
+
+  # GET /foods/1 or /foods/1.json
+  def show; end
+
+  # GET /foods/new
+  def new
+    @food = Food.new
+  end
+
+  # GET /foods/1/edit
+  def edit; end
+
+  # POST /foods or /foods.json
+  def create
+    @food = Food.new(food_params)
+
+    respond_to do |format|
+      if @food.save
+        format.html { redirect_to user_recipe_foods_path(params[:user_id]), notice: 'Food was successfully created.' }
+        format.json { render :show, status: :created, location: @food }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @food.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /foods/1 or /foods/1.json
+  def update
+    respond_to do |format|
+      if @food.update(food_params)
+        format.html { redirect_to user_recipe_path(params[:user_id]), notice: 'Food was successfully updated.' }
+        format.json { render :show, status: :ok, location: @food }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @food.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /foods/1 or /foods/1.json
+  def destroy
+    @food.destroy
+    respond_to do |format|
+      format.html { redirect_to user_recipe_path, notice: 'Food was successfully destroyed.' }
+      format.json { head :no_content }
+    end
```

Comment:
- [ ] @team kindly create a fail path/outcome for your destroy method like you did in your create and update methods

Reply:
Ok you right i fix it